mapflow/functional/view/data_catalog_view.py

```diff
@@ -511,6 +522,43 @@ def filter_catalog_table(self, name_filter: str = None):
             item_name = table.item(row, 1).data(Qt.DisplayRole)
             hide = bool(name_filter) and ((name_filter.lower() not in item_id.lower() and name_filter.lower() not in item_name.lower()))
             table.setRowHidden(row, hide)
+    
+    def show_processing_source(self, 
+                               provider: ProcessingParams):
+        """Switch to My imagery or user provider dialog when ckicking on toSourceButton in ProcessingDetails dialog."""
+        if provider.get("myImagery"):
```

Comment:
I can't understand how this provider.get() works, if provider is an instance of ProcessingParams. Let's have a call